docs/content/en/docs/concepts/eksafeatures.md

```diff
@@ -51,3 +51,19 @@ To learn more about Amazon EKS, see [Amazon Elastic Kubernetes Service](https://
 | Control plane pricing                       | Free to download, paid support subscription option  | Hourly pricing per cluster |
 | AWS Support                       | Additional annual subscription (per cluster) for AWS support | Basic support included. Included in paid AWS support plans (developer, business, and enterprise)  |
 ||||
+
+### Comparing Amazon EKS Anywhere to Amazon EKS on Outposts
+
+Like EKS Anywhere, Amazon EKS on Outposts provides a means of running Kubernetes clusters using EKS software and customer on-premises hardware.
+The main difference are that:
+
+* Amazon provides the hardware with Outposts, while most EKS Anywhere providers leverage the customer's own hardware.
+* Outposts have the option of having the EKS control plane run and be managed on the AWS cloud.
+
```

Comment:
Amazon EKS on Outposts is a regional AWS service that requires a consistent, reliable connection from the Outpost to the AWS Region. EKS Anywhere is a standalone software offering that can run entirely disconnected from AWS Cloud, including air-gapped environments.

Reply:
Done.